telosnetwork/teloscan
# #360 | Fix verify contract link behavior

# Fixes #360 

## Description
This PR updates the contract verification link behavior to prevent a blank page from loading when clicked. It also removes dead code.

## Test scenarios
- go to an unverified contract, e.g. https://deploy-preview-362--teloscan-stage.netlify.app/address/0xcc1390B993680B6a19608668E4012eEb8a0b4A8F
- click the red :warning: icon next to "Contract"
- in the modal that appears, click "Verify contract"
    - a new tab should open to sourcify.dev
    - the old tab should stay open to the address page

## Checklist:
